debezium-quarkus/pom.xml

```diff
@@ -0,0 +1,92 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>io.debezium</groupId>
+        <artifactId>debezium-parent</artifactId>
+        <version>1.1.0-SNAPSHOT</version>
+        <relativePath>../pom.xml</relativePath>
+    </parent>
+
+    <groupId>io.debezium.quarkus</groupId>
+    <artifactId>debezium-quarkus-parent</artifactId>
+    <packaging>pom</packaging>
+
+    <name>Debezium Quarkus</name>
+    <description>Debezium extensions for Quarkus</description>
+    <url>http://debezium.io</url>
```

Comment:
How comes we override this one (should be HTTPS btw.) and the others like encoding, which are inherited?

Reply:
Just a mistake on my part.